lua/nvim-tree/view.lua

```diff
@@ -187,6 +188,23 @@ function M.open(options)
   events._dispatch_on_tree_open()
 end
 
+local function grow()
+  local lines = vim.api.nvim_buf_get_lines(M.get_bufnr(), 0, -1, false)
```

Comment:
This includes the root folder. This results in a very wide window when you are in a deep path.